docs/samples/hub-state-import-data-sample.json

```diff
@@ -0,0 +1,925 @@
+{
+  "children": {
```

Comment:
Shouldn't this be encapsulated under a device centric element rather than at the top level?